assets/scss/1_base/4-buttons.scss

```diff
@@ -0,0 +1,27 @@
+.btn {
+  display: inline-block;
+  border: 0;
+  outline: 0;
+  box-shadow: none;
+  padding: 5px 10px;
+  cursor: pointer;
+  border-radius: 2px;
+  background: $gray;
+  transition: bacground 0.3s ease;
+  will-change: bacground;
+
+  &--primary {
```

Comment:
Personally, I'd avoid `&--primary` because it makes it difficult to find such styles when searching for `btn--primary`. Explicit code would be beneficial I think ;)

```
.btn { ... }

.btn--primary { ... } 

.btn--xs-full { @media { ... } } 
```